packages/insomnia-smoke-test/tests/app.test.ts

```diff
@@ -17,43 +18,44 @@ test('can send requests', async ({ app, page }) => {
   await app.evaluate(async ({ clipboard }, text) => clipboard.writeText(text), text);
 
   await page.click('button:has-text("Clipboard")');
-  await page.click('text=CollectionSmoke testsjust now');
 
-  await page.click('button:has-text("GETsend JSON request")');
-  await page.click('text=http://127.0.0.1:4010/pets/1Send >> button');
+  await page.click('_react=WorkspaceCard[workspace.name = "Smoke tests"]');
```

Comment:
at a minimum we would need to 
```suggestion
  await page.click(`_react=${WorkspaceCard.displayName}[workspace.name = "Smoke tests"]`);
```

or is it `.name`?  Well either way, we'd need to import the component, which means we'd first need to export it from `insomnia` which we don't do, today, yet (for no good reason).